superduperdb/base/datalayer.py

```diff
@@ -341,6 +347,11 @@ def _insert(
             if random.random() < s.CFG.fold_probability:
                 r['_fold'] = 'valid'
 
+        if auto_schema and self.cfg.auto_schema:
```

Comment:
Should we check if the `Document` instances have `Document.schema` set?

Reply:
Yes, I will handle it in the backend.

Reply:
Done